src/js/edu-benefits/1995/config/form.js

```diff
@@ -123,7 +124,7 @@ const formConfig = {
               'view:newService': {
                 type: 'boolean'
               },
-              toursOfDuty
+              toursOfDuty: fullSchema1995.properties.toursOfDuty
```

Comment:
This just seems less verbose than what we're doing for most of the properties we're pulling out of the form schemas and using once.

```js
const {
  someProp,
  anotherProp,
  moreProp,
  etc
} = fullSchema1234.properties;
...
```